### PR TITLE
add subkey to GITHUB_PATH

### PR DESCRIPTION
**Pull request by edwardmack**
## Changes

- Added entry to tests.yml to install subkey to GITHUB_PATH.
-
-

## Tests

```

```

## Checklist

- [x] I have read [CODE_OF_CONDUCT](https://github.com/ChainSafe/gossamer/blob/development/.github/CODE_OF_CONDUCT.md) and [CONTRIBUTING](https://github.com/ChainSafe/gossamer/blob/development/.github/CONTRIBUTING.md) 
- [x] I have provided as much information as possible and necessary
- [x] I have reviewed my own pull request before requesting a review
- [ ] All integration tests and required coverage checks are passing

## Issues

-

### CODE REVIEW

**Comment by edwardmack** [comment]
Since this install subkey to $HOME/.local/bin and sets that to $GITHUB_PATH, I'm not sure if we also need to copy it to <PROJECT_BASE>/bin?  Also, not sure if we still need `setSubkeyPath` function, or if it's already set since it's in $GITHUB_PATH?

**Comment by noot** [comment]
@edwardmack maybe try deleting the commited `bin/subkey` file and see if the rpc tests work? (i think you might also need to make a code change to get the CI to run, so maybe add a space or something somewhere..)

**Review by RyRy79261 — APPROVED** [review]

File: dot/rpc/modules/author.go [hunk 1]
```diff
@@ -185,7 +185,7 @@ func (cm *AuthorModule) SubmitExtrinsic(r *http.Request, req *Extrinsic, res *Ex
 	if cm.coreAPI.IsBlockProducer() {
 		hash := cm.txStateAPI.AddToPool(vtx)
 		*res = ExtrinsicHashResponse(hash.String())
-		cm.logger.Trace("submitted extrinsic", "tx", vtx, "hash", hash.String())
+		cm.logger.Trace("submitted extrinsic", "tx ", vtx, "hash", hash.String())
```
Comment (arijitAD) [inline dot/rpc/modules/author.go:188]: Revert this.

**Review by arijitAD — APPROVED** [review]

**Comment by arijitAD** [comment]
> @edwardmack maybe try deleting the commited `bin/subkey` file and see if the rpc tests work? (i think you might also need to make a code change to get the CI to run, so maybe add a space or something somewhere..)

We can also submit a blank commit to trigger CI. 
```
git commit --allow-empty -m "Trigger CI"
```

**Review by noot — APPROVED** [review]

**Review by noot — COMMENTED** [review]
can you remove the `bin/subkey` file?

**Comment by edwardmack** [comment]
`bin/subkey` has been deleted.

**Pull request merged**